tools/devdoc.md

```diff
@@ -0,0 +1,20 @@
+# Build and publishing
+
+In order for the Java Worker to get loaded into the runtime, today we use a NuGet to package the jars for deployment.
+
+## AppVeyor
+
+We're using App Veyor for CI. It builds the Java Core and Worker modules and then packages them into a NuGet for consumption by the Azure Functions Script Host.
+
+CI runs on every PR and nightly on `dev` and `master`
```

Comment:
Is it OK to assume that only the changes in `master` branch will be packed into the latest runtime.

Reply:
I did dev and master since master is usually the "released" bits and dev and the "to be released" bits. It's what the rest of the Functions repos do.

Reply:
Sure. So does that mean we may do experiments in `dev` branch, and the user won't notice that until we pushed them to `master`?

Reply:
This just produces CI builds for us, I didn't automate publishing, so users won't see anything we do unless we publish.